### PR TITLE
Astro 2/21 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/platform-variables.md [hunk 1]
```diff
@@ -21,39 +21,43 @@ If you need to set one of these variables for a particular use case, contact [As
...
-| `OPENLINEAGE_URL` | The URL for your Astro lineage backend. The destination for lineage metadata sent from external systems to the OpenLineage API. | `https://astro-<your-astro-base-domain>.datakin.com` |
-| `OPENLINEAGE_API_KEY` | Your OpenLineage API key | `<your-lineage-api-key>` |
+| Environment Variable                                               | Description                                                                                                                     | Value                                                      |
+| ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| `AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_TIMEOUT`         | The amount of time that the Celery executor waits for a response from the Celery backend before throwing an error.                           | `30`                                                       |
```
Comment (jwitz) [inline astro/platform-variables.md:26]: @goutamtadi1 Can you please review to ensure that these descriptions are accurate?

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,45 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
 
+## February 20, 2024
+
+### New worker types
+
+<HostedBadge/>
+
+Astro Hosted Deployments now support A120 and A160 workers, which include enough CPU and memory to handle the most resource-intensive tasks in your DAGs. See [Astro Hosted resource reference](resource-reference-hosted.md#deployment-resources) for more information about each worker type.
+
+### New platform variables to ensure reliability with the Celery executor
```
Comment (jwitz) [inline astro/release-notes.md:32]:
```suggestion
### New platform variables to improve Celery executor reliability
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,45 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+```
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_TIMEOUT=30
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_CONNECT_TIMEOUT=5
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_KEEPALIVE=True
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__RETRY_ON_TIMEOUT=True
+```
+
+For more information about each of these variables, see [Platform variables](platform-varaibles.md)
+
+### Ephemeral storage limit on schedulers
```
Comment (jwitz) [inline astro/release-notes.md:45]: @claassen Can you please review this release note for technical accuracy? Thanks!

**Review by claassen — APPROVED** [review]
Docs for new A120/A160 machines sizes look good to me

**Review by chomipi88 — APPROVED** [review]
LGTM for new worker types

File: astro/platform-variables.md [hunk 4]
```diff
@@ -21,39 +21,43 @@ If you need to set one of these variables for a particular use case, contact [As
...
-| `AWS_SECRET_ACCESS_KEY` | The key secret for accessing Astro's managed S3 bucket¹ | `<s3-aws-access-key-secret>` |
-| `INSTANCE_TYPE` | Provides the instance size of the node the DAG is scheduled on. | `(v1:metadata.labels['beta.kubernetes.io/instance-type'])` |
-| `OPENLINEAGE_URL` | The URL for your Astro lineage backend. The destination for lineage metadata sent from external systems to the OpenLineage API. | `https://astro-<your-astro-base-domain>.datakin.com` |
-| `OPENLINEAGE_API_KEY` | Your OpenLineage API key | `<your-lineage-api-key>` |
+| Environment Variable                                               | Description                                                                                                                     | Value                                                      |
+| ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
```
Comment (lzdanski) [inline astro/platform-variables.md:25]: Screenshot 2024-02-21 at 1 01 58 PM
Getting some funky auto-formatting

**Review by lzdanski — APPROVED** [review]
A couple of minor things - I don't think one of them is fixable in the scope of this PR, but seeing some funky table formatting.

File: astro/release-notes.md [hunk 5]
```diff
@@ -21,6 +21,41 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+Astro Deployments now have the following environment variables set by default. This is to ensure that the Celery executor doesn't freeze when it attempts to connect with a Pod in the Celery backend that has unexpectedly terminated.
+
+```
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_TIMEOUT=30
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_CONNECT_TIMEOUT=5
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__SOCKET_KEEPALIVE=True
+AIRFLOW__CELERY_BROKER_TRANSPORT_OPTIONS__RETRY_ON_TIMEOUT=True
+```
+
+For more information about each of these variables, see [Platform variables](platform-variables.md)
```
Comment (jwitz) [inline astro/release-notes.md:43]:
```suggestion
For more information about each of these variables, see [Platform variables](https://docs.astronomer.io/astro/platform-variables)
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -21,6 +21,41 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## February 21, 2024
+
+### New worker types
+
+<HostedBadge/>
+
+Astro Hosted Deployments now support A120 and A160 workers, which include enough CPU and memory to handle the most resource-intensive tasks in your DAGs. See [Astro Hosted resource reference](resource-reference-hosted.md#deployment-resources) for more information about each worker type.
```
Comment (jwitz) [inline astro/release-notes.md:30]:
```suggestion
Astro Hosted Deployments now support A120 and A160 workers, which include enough CPU and memory to handle the most resource-intensive tasks in your DAGs. See [Astro Hosted resource reference](https://docs.astronomer.io/astro/resource-reference-hosted#deployment-resources) for more information about each worker type.
```

**Pull request merged**